fs/layer/layer.go

```diff
@@ -345,7 +345,11 @@ func (r *Resolver) Resolve(ctx context.Context, hosts []docker.RegistryHost, ref
 	if err != nil {
 		return nil, fmt.Errorf("failed to read layer: %w", err)
 	}
-	disableXattrs := getDisableXAttrAnnotation(sociDesc)
+	disableXattrs := false
+	if !r.config.ForceGetXAttrs {
+		fmt.Println("xattrs checking")
```

Comment:
I don't see any other instances of `fmt.Println` in this file – should this be a log statement? can it be a little more verbose?

Reply:
Debugging log for myself that I forgot to remove, whoops